certif/app/controllers/authenticated/team.js

```diff
@@ -15,6 +15,24 @@ export default class Team extends Controller {
     );
   }
 
+  get membersSelectOptionsSortedByLastName() {
+    return this.model.members
+      .toArray()
+      .sort((a, b) => {
+        const lastName1 = a.lastName.toLowerCase();
+        const lastName2 = b.lastName.toLowerCase();
+
+        if (lastName1 < lastName2) {
+          return -1;
+        }
+        if (lastName1 > lastName2) {
+          return 1;
+        }
+        return 0;
+      })
```

Comment:
On peut comparer directement des chaines avec localCompare. 
Avec l'option `sensitivity: 'base'` il n'est pas sensible à la casse ni aux accents (`A = à = a != b`)
```suggestion
   .sort((lastName1, lastName2) => lastName1.localeCompare(lastName2, 'fr-FR', {sensitivity: 'base'}))
```